### DISCUSSION TITLE
Company name with special characters

### DISCUSSION THREAD

**Discussion by LongnoseRob** [pymeasure/pymeasure#463] [General]
I might start soon on adding an instrument made by Rohde&Schwarz, 

As far as I can see there is no open PR for an R&S product already also there is no folder for R&S under instruments yet.

To be on the safe side I will start my implementation witn an folder named `rs.`
Or would `rohde_schwarz` be the better folder name?

Modern OS would probably be ok with the _&_ in a folder-name, but I would like to stick to a basic folder-name without any special characters.

any opinions?

**Comment by bilderbuchi** [comment]
Thanks for thinking ahead! Yes, please keep python-legal identifiers. Following other multi-word company names in the repo (which just concatenate), I think `rohdeschwarz` would probably be best (also neatly sidestepping if the `&` should be `und` or `and`).

**Reply by LongnoseRob** [reply]
I agree with you `rohdeschwarz` is probably the best,
The company name is read as Rohde und Schwarz, named after the two founding engineers.

I will use the foldername of `rohdeschwarz` for the moment
